components/engine_rocks/src/perf_context_impl.rs

```diff
@@ -45,6 +46,18 @@ macro_rules! observe_perf_context_type {
     };
 }
 
+lazy_static! {
+    /// Default perf flags for a write operation.
+    static ref DEFAULT_WRITE_PERF_FLAGS: PerfFlags = PerfFlag::WriteWalTime
```

Comment:
What's the difference with `EnableTime`

Reply:
Not much.
One difference is that we don't record time for other operations like RocksDB read in the same thread. If necessary, we can choose what we need later.
And because we are going to record durations about reading in the read pool with PerfFlags. This change will make it more consistent in the whole workspace.